### PR TITLE
Re-add GitHub Action to comment on someone's first pull request

### LINKED ISSUE #1965: Fix the GitHub Action that comments on pull requests
In #1956, a change was made to switch the GitHub Action that comments on pull requests, based off of what was implemented in Astropy.  However, we're now getting an error from that which we need to fix: 
```
SyntaxError: Unexpected identifier
    at new AsyncFunction (<anonymous>)
    at callAsyncFunction (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:13356:16)
    at main (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:13452:26)
    at Module.858 (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:13429:1)
    at __webpack_require__ (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:24:31)
    at startup (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:43:19)
    at /home/runner/work/_actions/actions/github-script/v6/dist/index.js:49:18
    at Object.<anonymous> (/home/runner/work/_actions/actions/github-script/v6/dist/index.js:52:10)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
Error: Unhandled error: SyntaxError: Unexpected identifier
```
It would have been helpful if the error message had said what the unexpected identifier was, but alas!

### PR DESCRIPTION
**Pull request by namurphy**
This PR adds back and updates a GitHub Action that adds a comment when someone makes their first pull request to PlasmaPy.  We had tentatively removed it because it ran into some problems (discussed in #2044).

In the long term, I'm thinking it would be worthwhile to change the checklist (which we haven't gotten into the habit of using...) into a bullet point list of code/doc/test tips.  I also want to add a link that goes directly to the documentation preview, since it's now necessary to navigate an extra step.  I don't think I'll do that in this PR, though.

Closes #1965.  (along with #2044)

### CODE REVIEW

File: .github/workflows/new-pull-request.yml [hunk 1]
```diff
@@ -5,6 +5,30 @@ on:
...
+        issues: write
+        pull-requests: write
+      steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: >
+            Thank you for submitting your first pull request (PR) to PlasmaPy!  We invite you to check out our [**Code of Conduct**](https://docs.plasmapy.org/en/stable/CODE_OF_CONDUCT.html) and [**contributor guide**](https://docs.plasmapy.org/en/latest/contributing/index.html). 🥳
+
+            Please **add yourself to the list of contributors** in [docs/about/credits.rst](https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/about/credits.rst) to make sure that you get credit for your contribution. 🔖
```
Comment (pheuer) [inline .github/workflows/new-pull-request.yml:24]: For the sake of new contributors, maybe specify that this means to modify that file within you PR

File: .github/workflows/new-pull-request.yml [hunk 2]
```diff
@@ -5,6 +5,30 @@ on:
...
+
+            If this PR is marked as ready for review, someone should stop by soon to provide a code review and offer suggestions. ✅ If you'd like more time beforehand 🕰️, please [convert this PR into a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) and later on [mark it as ready for review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review). If you don't get a review within ∼2–3 weekdays, please feel free to give us a reminder.
+
+            We thank you once again! 🪩
```
Comment (pheuer) [inline .github/workflows/new-pull-request.yml:30]: Emoji not rendering?

**Review by pheuer — COMMENTED** [review]
Looks good - a couple minor notes

File: .github/workflows/new-pull-request.yml (hunk 2 above)
Comment (namurphy) [reply .github/workflows/new-pull-request.yml:30]: Looks like the disco ball emoji was added in unicode 14.0 which is fairly recent. I'll change it to a celebratory emoji that was around earlier.  It's still too early for a disco renaissance, I suppose!

File: .github/workflows/new-pull-request.yml (hunk 1 above)
Comment (namurphy) [reply .github/workflows/new-pull-request.yml:24]: Very good clarification to include!

**Comment by namurphy** [comment]
@pheuer — thank you for the review!  Did you have any other thoughts about this PR or is it ready to merge?

**Pull request merged**